src/Compilers/CSharp/Portable/Parser/Lexer_StringLiteral.cs

```diff
@@ -242,11 +250,23 @@ internal static SyntaxToken RescanInterpolatedString(InterpolatedStringExpressio
                 interpolatedString.GetLastToken().GetTrailingTrivia());
         }
 
+        internal enum InterpolatedStringKind
+        {
+            /// <summary>
+            /// Normal interpolated string that just starts with $"
+            /// </summary>
+            Normal,
+            /// <summary>
+            /// Verbatim interpolated string that starts with $@" or @$"
+            /// </summary>
+            Verbatim,
+        }
```

Comment:
this change really helps out the raw-string-literal branch.  there we have 4 different kinds, so moving from a bool to an enum is beneficial.